manifests/hiera.pp

```diff
@@ -1,10 +1,8 @@
 #
 class riemann::hiera (
   $prefix = ''
-)
-  {
+) {
```

Comment:
do you know how to find which linter rule this is and disable it globally ?
Sorry, but this is the kind of thing that gets me...

Reply:
Caused by:

```
ERROR: there should be a single space before an opening brace on line 5 (check: manifest_whitespace_opening_brace_before)
```

Not aligning opening and closing braces looks weird, would you not prefer?

```puppet
class riemann::hiera (
  # ...
)
{
  # ...
}
```

to

```puppet
class riemann::hiera (
  # ...
)
  {
  # ...
}
```

Reply:
I do, I just didn't see it.